src/java.base/share/classes/java/util/concurrent/StructuredTaskScopeImpl.java

```diff
@@ -0,0 +1,442 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
No changes in 2025?

Reply:
It was created in 2024, hasn't had any changes until the rename from Config -> Configuration.